Localization/extended_kalman_filter/extended_kalman_filter.py

```diff
@@ -147,9 +147,9 @@ def plot_covariance_ellipse(xEst, PEst):  # pragma: no cover
     b = math.sqrt(eigval[smallind])
     x = [a * math.cos(it) for it in t]
     y = [b * math.sin(it) for it in t]
-    angle = math.atan2(eigvec[bigind, 1], eigvec[bigind, 0])
-    rot = np.array([[math.cos(angle), math.sin(angle)],
-                    [-math.sin(angle), math.cos(angle)]])
+    angle = math.atan2(eigvec[1, bigind], eigvec[0, bigind])
+    rot = np.array([[math.cos(angle), -math.sin(angle)],
```

Comment:
Could you please use scipy `Rot`?
https://github.com/AtsushiSakai/PythonRobotics/blob/468be047aa5975265cbdc5b89740bc6b61079a9e/Localization/particle_filter/particle_filter.py#L192

Reply:
Ok, I've changed the rotation matrix to `scipy.spatial.transform.Rotation.from_euler()` in [7a49ca9](https://github.com/AtsushiSakai/PythonRobotics/pull/407/commits/7a49ca964dea89d390078f20fd453a9d12a70a23)